### PR TITLE
launch Microsoft Edge with specified args

### PR DESCRIPTION
**Pull request by gp5251**
This PR adds an option to enable cors when launching instance. 

image

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -400,11 +400,16 @@ export async function launchBrowser(browserPath: string, port: number, targetUrl
...
 
     const headless: boolean = isHeadlessEnabled();
+    const corsEnabled: boolean = isCorsEnabled();
 
     if (userDataDir) {
         args.unshift(`--user-data-dir=${userDataDir}`);
     }
 
+    if (corsEnabled) {
+        args.unshift('--disable-web-security');
```
Comment (hkal) [inline src/utils.ts:410]: `--disable-web-security` disables same-origin policy, which isn't the same as CORS iirc. Does launching the browser with this switch address your usecase? Additionally, if CORS is enabled we don't want to pass this switch, so the check should be inversed.

**Review by hkal — CHANGES_REQUESTED** [review]
Thanks for the PR. The team is planning on generalizing passing browser cmdline switches so we don't have to add new extension settings for each one, since there are many. Can you update the PR to work with an extension setting like:
```
"browserArgs" : [
  "--disable-web-security",
  "--user-data-dir=/some/custom/path"
]
```

File: src/utils.ts (hunk 1 above)
Comment (gp5251) [reply src/utils.ts]: yes, `--disable-web-security` works for me.
ok, i changed a bit, using an array as browserArgs now.  should meet the feature.

Comment (gp5251) [reply src/utils.ts]: image

Comment (hkal) [reply src/utils.ts]: Great. Update the title of the PR to reflect the new work and I'll approve :)

Comment (gp5251) [reply src/utils.ts]: updated

**Review by hkal — APPROVED** [review]
LGTM

**Pull request merged**